img/u/ubc.ca.svg

```diff
@@ -0,0 +1,12 @@
+  <g data-name="Layer">
+    <clipPath id="a">
+      <path transform="matrix(1 0 0 -1 0 198.055)" d="M0 198.055h198.333V0H0Z"/>
+    </clipPath>
+    <g clip-path="url(#a)" fill="#00063e">
+      <path d="m99.166 87.58 5.279 38.287 17.346-28.985-7.36 31.882 18.7-17.891-9.739 23.619 15.56-11.611c-7.243 19.709-13.553 30.758-39.785 39.74-26.232-8.982-32.543-20.031-39.785-39.74l15.559 11.61-9.739-23.618 18.7 17.89-7.36-31.881 17.347 28.985 5.278-38.287ZM71.687 74.751c-7.948 0-12.695-4.253-12.695-11.4V46.243h6.59v16.84c0 4.151 2.183 6.434 6.156 6.434 4.074 0 6.234-2.187 6.234-6.317V46.243h6.633v16.995c0 7.103-4.949 11.513-12.918 11.513M99.414 74.278h-9.202V46.244h9.339c6.827 0 10.589 2.65 10.589 7.445 0 2.524-1.316 4.466-3.828 5.655 3.368 1.189 5.079 3.414 5.079 6.629 0 5.359-4.251 8.305-11.977 8.305m-2.836-4.682h2.836c2.563 0 5.609-.654 5.609-3.78 0-3.22-3.097-3.905-5.703-3.905h-2.742Zm0-12.09h2.395c3.206 0 4.98-1.203 4.98-3.384 0-2.145-1.546-3.191-4.72-3.191h-2.655ZM129.847 74.751c-9.074 0-14.933-5.666-14.933-14.453 0-8.684 6.073-14.523 15.112-14.523 3.352 0 6.172.66 8.622 2.02l.214.12-.984 5.707-.434-.289c-2.406-1.615-4.695-2.36-7.195-2.36-5.391 0-8.613 3.474-8.613 9.29 0 5.71 3.332 9.254 8.702 9.254 2.544 0 5.047-.764 7.653-2.332l.43-.258.929 5.325-.207.12c-2.675 1.532-5.976 2.38-9.296 2.38"/>
+      <path d="M99.167 168.559c-30.97-8.773-45.75-29.22-45.75-58.367V41.196h91.5v68.996c0 29.148-14.782 49.594-45.75 58.367m-49.5-131.113v73.215c0 30.77 16.742 53.048 49.5 61.785 32.757-8.737 49.5-31.015 49.5-61.785V37.446Z"/>
+      <path d="M99.168 84.207c6.902 0 7.835 2.8 13.395 2.766 5.54.034 6.487-2.766 13.39-2.766 6.902 0 7.851 2.8 13.395 2.766v-6.428c-5.544.037-6.493-2.729-13.395-2.729-6.903 0-7.85 2.766-13.39 2.73-5.56.036-6.493-2.73-13.395-2.73-6.903 0-7.836 2.766-13.395 2.73-5.539.036-6.488-2.73-13.39-2.73-6.903 0-7.856 2.766-13.395 2.73v6.427c5.54.034 6.492-2.766 13.395-2.766 6.902 0 7.851 2.8 13.39 2.766 5.56.034 6.492-2.766 13.395-2.766M103.541 95.823c3.33.797 4.915 2.318 9.022 2.292 5.538.035 6.487-2.766 13.39-2.766 6.901 0 7.851 2.8 13.395 2.766v-6.428c-5.544.037-6.494-2.728-13.396-2.728-6.901 0-7.85 2.765-13.39 2.728-4.462.03-5.947-1.745-9.927-2.447ZM85.774 98.115c4.106.026 5.69-1.495 9.018-2.291l.907-6.584c-3.98.702-5.464 2.477-9.925 2.447-5.54.037-6.49-2.728-13.391-2.728-6.902 0-7.855 2.765-13.394 2.728v6.428c5.539.035 6.492-2.766 13.394-2.766s7.852 2.8 13.39 2.766M105.142 107.433c1.883.647 3.33 1.438 5.566 1.718l3.859-6.446c-.603.082-1.262.128-2.004.123-3.819.026-5.455-1.27-8.343-2.086ZM122.812 106.721c.902-.142 1.928-.23 3.14-.23 6.903 0 7.853 2.8 13.396 2.766v-6.428c-5.543.037-6.493-2.729-13.395-2.729-.583 0-1.121.022-1.626.058ZM75.522 106.721l-1.516-6.563a22.686 22.686 0 0 0-1.623-.057c-6.902 0-7.855 2.765-13.394 2.728v6.428c5.539.035 6.492-2.766 13.395-2.766 1.21 0 2.236.088 3.138.23M87.625 109.152c2.237-.28 3.684-1.07 5.566-1.718l.922-6.69c-2.886.814-4.522 2.11-8.339 2.085a13.96 13.96 0 0 1-2.008-.125Z"/>
+    </g>
+  </g>
+</svg>
```

Comment:
I suggest you run the SVGs through [SVGOMG](https://jakearchibald.github.io/svgomg/) first as it minimized the SVGs drastically.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="#00063e" viewBox="0 0 99 135"><path d="m49.5 50.13 5.27 38.3 17.35-29-7.36 31.88 18.7-17.89-9.74 23.62 15.56-11.6c-7.24 19.7-13.55 30.75-39.78 39.73-26.23-8.98-32.54-20.03-39.79-39.74l15.56 11.61-9.73-23.62 18.7 17.9-7.37-31.88 17.35 28.98 5.28-38.29zM22.02 37.3c-7.95 0-12.7-4.25-12.7-11.4V8.8h6.6v16.84c0 4.15 2.18 6.43 6.15 6.43 4.07 0 6.23-2.19 6.23-6.32V8.8h6.64v17c0 7.1-4.95 11.5-12.92 11.5m27.73-.47h-9.2V8.8h9.33c6.83 0 10.6 2.65 10.6 7.44 0 2.53-1.33 4.47-3.84 5.66 3.37 1.19 5.08 3.41 5.08 6.63 0 5.36-4.25 8.3-11.97 8.3m-2.84-4.68h2.84c2.56 0 5.6-.65 5.6-3.78 0-3.22-3.09-3.9-5.7-3.9h-2.74zm0-12.09h2.4c3.2 0 4.97-1.2 4.97-3.38 0-2.15-1.54-3.2-4.71-3.2H46.9zm33.27 17.25c-9.08 0-14.94-5.67-14.94-14.46 0-8.68 6.08-14.52 15.12-14.52 3.35 0 6.17.66 8.62 2.02l.21.12-.98 5.7-.44-.28a12.47 12.47 0 0 0-7.19-2.36c-5.39 0-8.6 3.47-8.6 9.29 0 5.7 3.32 9.25 8.7 9.25 2.53 0 5.04-.76 7.64-2.33l.43-.26.93 5.33-.2.12a18.93 18.93 0 0 1-9.3 2.38ZM49.5 131.11c-30.97-8.77-45.75-29.22-45.75-58.37V3.75h91.5v69c0 29.14-14.79 49.59-45.75 58.36M0 0v73.21c0 30.77 16.74 53.05 49.5 61.79C82.25 126.26 99 103.98 99 73.21V0ZM49.5 46.76c6.9 0 7.83 2.8 13.4 2.77 5.53.03 6.48-2.77 13.38-2.77s7.85 2.8 13.4 2.77V43.1c-5.55.04-6.49-2.73-13.4-2.73-6.9 0-7.85 2.77-13.39 2.73-5.56.04-6.49-2.73-13.39-2.73s-7.83 2.77-13.4 2.73c-5.53.04-6.48-2.73-13.38-2.73s-7.86 2.77-13.4 2.73v6.43c5.54.03 6.5-2.77 13.4-2.77s7.85 2.8 13.39 2.77c5.56.03 6.49-2.77 13.4-2.77m4.36 11.62c3.33.8 4.92 2.31 9.02 2.29 5.54.04 6.5-2.77 13.4-2.77s7.84 2.8 13.4 2.77v-6.43c-5.56.04-6.5-2.73-13.4-2.73s-7.86 2.77-13.4 2.73c-4.46.03-5.94-1.74-9.92-2.45zm-17.76 2.29c4.1.03 5.69-1.5 9.02-2.3l.9-6.58c-3.98.7-5.46 2.48-9.92 2.45-5.54.04-6.5-2.73-13.4-2.73s-7.85 2.77-13.39 2.73v6.43c5.54.04 6.5-2.77 13.4-2.77s7.85 2.8 13.39 2.77m19.36 9.31c1.88.65 3.33 1.44 5.57 1.72l3.86-6.44c-.6.1-1.27.12-2 .12-3.83 0-5.46-1.27-8.35-2.09zm17.67-.7c.9-.15 1.93-.24 3.14-.24 6.9 0 7.86 2.8 13.4 2.77v-6.43c-5.54 0-6.49-2.73-13.4-2.73-.58 0-1.12 0-1.62.1zm-47.28 0-1.52-6.57a22.69 22.69 0 0 0-1.62-.1c-6.9 0-7.86 2.77-13.4 2.73v6.43c5.54 0 6.5-2.77 13.4-2.77 1.2 0 2.23.1 3.14.23m12.1 2.43c2.24-.28 3.68-1.07 5.56-1.72l.93-6.69c-2.89.82-4.53 2.11-8.34 2.1a13.96 13.96 0 0 1-2.01-.14Z"/></svg>
```